src/controllers/coops.js

```diff
@@ -0,0 +1,58 @@
+import db from '../models';
+const {coops} = db;
+
+class Coops {
+
+	// creating a function for add cooperative
+	static async createCoop(req,res){
+		// destructing data from body
+		const {coopName,coopLocation,coopLicense,coopReaderName,coopReaderId,coopEmail} = req.body;
+
+		// try and catch to find if not exist create new coop
+		try{
+			// find if exist
+			const findOne = await coops.findOne({where: {coopLicense}});
+			if (findOne){
+				res.status(403).send({
+					message:'The cooperative with that tin exists',
+					coopname:findOne.coopName
+				})
+			}
+			else{
+				const createCoop = await coops.create({
+					coopName,
+					coopLocation,
```

Comment:
care about the identation

Reply:
man i always take care of indents but i don't know where is the mistake